src/tools/OptionsSchema.json

```diff
@@ -406,7 +406,11 @@
         },
         "targetArchitecture": {
           "type": "string",
-          "description": "[Only supported in local macOS debugging] The architecture of the debuggee. This will automatically be detected unless this parameter is set. Allowed values are x86_64 or arm64."
+          "enum": [
+            "x86_64",
+            "arm64"
+          ],
+          "description": "The architecture of the debuggee. This will automatically be detected unless this parameter is set. Allowed values are x86_64 or arm64."
```

Comment:
I would suggest also adding: `This value is ignored on Linux.` at the end of the description.

Reply:
Changes to `OptionsSchema.json` also require you to run `gulp generateOptionsSchema`.

See https://github.com/OmniSharp/omnisharp-vscode/blob/master/src/tools/README.md

Reply:
I have added `This value is ignored on Linux.` to the end of the description and re-generated the schema.